SharedProjects/GltfExport.Entities/GLTFMaterial.cs

```diff
@@ -63,7 +67,7 @@ public bool ShouldSerializeemissiveTexture()
 
         public bool ShouldSerializeemissiveFactor()
         {
-            return (this.emissiveFactor != null && !this.emissiveFactor.SequenceEqual(new float[] { 0f, 0f, 0f }));
+            return (this.emissiveFactor != null && !this.emissiveFactor.SequenceEqual(zeros3));
```

Comment:
I reccomend using tools.isAlmostEqualTo() instead of doing floating point equivalency.

Reply:
I agree, Then we use the MathUtilities wich is already used in GLTFLight, then reduce the dependencies. 
and add 
```
        public static bool IsAlmostEqualTo(this float[] array, float value, float epsilon)
        {
            return ! array.Any( v=> Math.Abs(v-value) > epsilon);
        }
```
into the game
ok ?

Reply:
Sounds good, thanks